build-packages/merge-changelogs/index.ts

```diff
@@ -25,26 +28,20 @@ interface ContentByVersion {
 }
 
 function getPackageName(changelog: string): string {
-  // Get package name without the @sap-cloud-sdk scope
-  const packageNameRegex = /^# @sap-cloud-sdk\/(?<packageName>.+)$/;
-  const firstLine = changelog.split('\n')[0];
-  const packageNameMatch = firstLine.match(packageNameRegex);
-
-  if (!packageNameMatch?.groups?.packageName) {
-    throw new Error('Changelog should have package name on first line');
-  }
-
-  return packageNameMatch.groups.packageName;
+  return changelog.split('\n')[0].split('/')[1];
 }
 
 function splitByVersion(changelog: string): ContentByVersion[] {
-  return changelog.split('\n## ').map(h2 => {
-    const [version, ...content] = h2.split('\n');
-    return {
-      version,
-      content: content.join('\n').trim()
-    };
-  });
+  return changelog
+    .split('\n## ')
+    .slice(1)
```

Comment:
Surprised this didn't cause issues before, I guess it returned an object looking like `{ version: '', content: '' }`